### PR TITLE
added visualization for min and max transforms

### PR DESCRIPTION
**Pull request by marinabar**
## What this does
Allows to visualize maximum and minimum transformations that could be applied to your dataset given a specified min_max range. Saves three additional files`min.png`, `mean.png`, and `max.png` for each transform.

## How it was tested
Unit test added : `test_visualize_image_transforms.py`

`pytest tests/test_visualize_image_transforms.py`

## How to checkout & try? (for the reviewer)
You can run the script on any config parameters. For example

`python lerobot/scripts/visualize_image_transforms.py training.image_transforms.contrast=(0.5, 0.5)`

- - -
This change is [Reviewable](https://reviewable.io/reviews/huggingface/lerobot/271)

### CODE REVIEW

File: lerobot/scripts/visualize_image_transforms.py [hunk 1]
```diff
@@ -118,6 +119,25 @@ def save_config_single_transforms(cfg, original_frame, output_dir):
...
+
+        # Apply max transformation
+        kwargs = {
+            f"{transform}_weight": cfg[f"{transform}"].weight,
+            f"{transform}_min_max": (max_value, max_value),
+        }
+        tf = get_image_transforms(**kwargs)
+        transformed_frame = tf(original_frame)
+        to_pil(transformed_frame).save(output_dir_single / "max.png", quality=100)
+
```
Comment (aliberts) [inline lerobot/scripts/visualize_image_transforms.py:140]: Could you add this one as well?
```suggestion
        # Apply mean transformation
        mean_value = (min_value + max_value) / 2
        kwargs = {
            f"{transform}_weight": cfg[f"{transform}"].weight,
            f"{transform}_min_max": (mean_value, mean_value),
        }
        tf = get_image_transforms(**kwargs)
        transformed_frame = tf(original_frame)
        to_pil(transformed_frame).save(output_dir_single / "mean.png", quality=100)
```

**Review by aliberts — CHANGES_REQUESTED** [review]
Thanks!
Left one addition for you.
Also, can you add a test for this script?
The test would be:
- 1. Run the script and ensure there are no errors
- 2. Check that the files and folders have been created (in `outputs/image_transforms`)

File: lerobot/scripts/visualize_image_transforms.py (hunk 1 above)
Comment (marinabar) [reply lerobot/scripts/visualize_image_transforms.py:140]: f98c9a71e2cc679fb43cbe50fd6181ab22a10493 done

**Review by aliberts — APPROVED** [review]
LGTM, thanks!

**Pull request merged**